addons/website/tests/test_performance.py

```diff
@@ -126,12 +126,13 @@ def test_11_perf_sql_img_controller(self):
         url = '/web/image/res.users/2/image_256'
         select_tables_perf = {
             'base_registry_signaling': 1,
+            'res_company': 1,
             'res_users': 2,
             'res_partner': 1,
             'ir_attachment': 2,
         }
-        self._check_url_hot_query(url, 6, select_tables_perf)
-        self.assertEqual(self._get_url_hot_query(url, cache=False), 6)
+        self._check_url_hot_query(url, 7, select_tables_perf)
+        self.assertEqual(self._get_url_hot_query(url, cache=False), 7)
```

Comment:
I still need to investigate why we have extra queries here.

Reply:
This test tries to access the image of a user that its read access is allowed. In the previous version check access was done via `_find_record_check_access` in website. In that process reading the only field from the self (user) was done by `self.sudo().website_published` and at the end it was returning `self.sudo()`.
In this version, since we're checking the read access to the record first it doesn't go through the extra check on website and the record is not sudo anymore.
The extra query is happening on `res.company` (`_compute_parent_ids`) for checking the read access to the record.